src/newSeed/seedConfig.ts

```diff
@@ -99,4 +100,24 @@ export class SeedConfig implements ISeedConfig {
   public seedDetails = {
     geoBlock: false,
   } as ISeedDetails;
+
+  clearState(): void {
+    this.general = {
```

Comment:
Duplicating code is not a good design.  Make the constructor call this function.